tokenizer.js

```diff
@@ -197,7 +199,7 @@
 				limit = options.numToSuggest || 1000,
 				list = [];
 				for (i = 0; word && i < words.length && list.length < limit; i++) {
-					if (!words[i].match(re1)) { continue; }
+					if (!words[i].match(re1) && !(options.allowFullSearch && word == options.fullSearchChar)) { continue; }
```

Comment:
I think I misunderstood this feature. So the goal here is to show 5 suggestions if the user types a `*`, right?

It might be better to have a `minChars` option (default = 1) determining how many characters need to be in the word before a dropdown appears. That does two things: (1) it helps with really long lists, and (2) with `minChars = 0`, we can automatically show suggestions when the input gets `focus`, which is more discoverable and friendly for new users.

There's an example of a similar implementation [here](https://github.com/graphiq/TagSelect.js/blob/master/src/Autocomplete.js#L26).